punishments.js

```diff
@@ -1486,11 +1483,12 @@ Punishments.isRoomBanned = function (user, roomid) {
  * options.publicOnly will make this only return public room punishments.
  * options.checkIps will also check the IP of the user for IP-based punishments.
  *
- * @param {User | string} user
+ * @param {User? | string} user
```

Comment:
It's a bad idea to change a call signature in a way that would obviously be a bug if actually used that way.

Just make a temporary variable...